### DISCUSSION TITLE
Calling CQ (without it answering random calls)

### DISCUSSION THREAD

**Discussion by TF1979** [N0BOY/FT8CN#110] [Q&A]
Hello,
I am trying to get my setup for SOTA FT8 working, using an Android 12 phone, Digirig Nano and FT-817ND.

I have not yet figured out how to get the app to call CQ, and all I find are videos of people answering CQ. I put SOTA in the modifier, then go to the 'Calling' tab: If I tap CQ I now get '6 CQ SOTA 2M0TFF IO85' in the lower bar. I have no idea why there is a 6 present, nor what it represents. There is also no clarity on how to then get the app to call this CQ. What I found yesterday on a summit was that the app would then respond randomly to a CQ call decoded which was at -14, and very unlikely to hear a 2.5w response.

Once I feel confident in my workflow I would be very happy to contribute a cheat-sheet style manual to the community.

73

**Comment by puntofisso** [comment]
It took a while for me to understand too. You need to push the megaphone button in the top right to enable TX.
